stdlib/inspect.pyi

```diff
@@ -259,7 +259,9 @@ class Arguments(NamedTuple):
     varkw: str | None
 
 def getargs(co: CodeType) -> Arguments: ...
-def getargspec(func: object) -> ArgSpec: ...
+
+if sys.version_info < (3, 11):
+    def getargspec(func: object) -> ArgSpec: ...
```

Comment:
`ArgSpec` no longer exists on 3.11. Its definition should be put inside an if statement too.